api/build.sh

```diff
@@ -32,13 +35,13 @@ set -x
 export ENV=local
 pwd=`pwd`
 
-rm -rf output && mkdir -p output/conf && mkdir -p output/dag-to-lua
+rm -rf output && mkdir -p output/conf && mkdir -p ./api/cmd/dag-to-lua
```

Comment:
oh, why change `output` to `api/cmd` ?

Reply:
for two reasons :) 
1. we cant embed an empty directory.
2. the embedded directory must be inside the package directory or subdirectory. for `// go:embed` to works.
>Go source files that import "embed" can use the //go:embed directive to initialize a variable of type string, []byte, or FS with the contents of files read from the package directory or subdirectories at compile time.


[reference](https://golang.org/pkg/embed/)